models/yield_yak/avalanche_c/yield_yak_avalanche_c_trades.sql

```diff
@@ -52,31 +54,31 @@ SELECT
     END as token_pair, 
     dexs.token_bought_amount_raw / power(10, erc20a.decimals) as token_bought_amount, 
     dexs.token_sold_amount_raw / power(10, erc20b.decimals) as token_sold_amount, 
-    CAST(dexs.token_bought_amount_raw AS DECIMAL(38,0)) AS token_bought_amount_raw, 
-    CAST(dexs.token_sold_amount_raw AS DECIMAL(38,0)) AS token_sold_amount_raw, 
+    CAST(dexs.token_bought_amount_raw AS UINT256) AS token_bought_amount_raw, 
+    CAST(dexs.token_sold_amount_raw AS UINT256) AS token_sold_amount_raw, 
     COALESCE(
         dexs.amount_usd, 
         (dexs.token_bought_amount_raw / power(10, p_bought.decimals)) * p_bought.price, 
         (dexs.token_sold_amount_raw / power(10, p_sold.decimals)) * p_sold.price
     ) as amount_usd, 
     dexs.token_bought_address, 
     dexs.token_sold_address, 
-    tx.from as taker,
+    tx."from" as taker,
     dexs.maker, 
     dexs.project_contract_address, 
     dexs.tx_hash, 
-    tx.from as tx_from, 
-    tx.to AS tx_to, 
+    tx."from" as tx_from, 
+    tx."to" AS tx_to, 
```

Comment:
Here you don't need the "" since `to` isn't a special keyword

Reply:
oke ser.